### PR TITLE
Multi part premium onboarding

### PR DESCRIPTION
**Pull request by maxxcrawford**

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -460,11 +462,36 @@ onboarding-premium-domain-description = With a custom domain, you can make your
 onboarding-premium-reply-title = Reply to emails without giving away your real address
 onboarding-premium-reply-description = Need to reply to emails sent to an alias? Just reply as normal — your alias will still protect your email address.
 
+## Multi-part Premium Onboarding - This is a walk-through onboarding expereince
+multi-part-onboarding-premium-welcome-headline = Welcome to { -brand-name-relay-premium }
+multi-part-onboarding-premium-welcome-subheadline = Now you can control what hits your inbox, one email at a time.
+multi-part-onboarding-premium-welcome-title = Control what emails you get:
+multi-part-onboarding-premium-welcome-description = { onboarding-premium-control-description }
+multi-part-onboarding-premium-domain-headline = Custom domain for sharing aliases
+multi-part-onboarding-premium-domain-subheadline = { multi-part-onboarding-premium-welcome-subheadline }
```
Comment (maxxcrawford) [inline en/app.ftl:471]: This is reused from above. This can be removed in favor of just using the same ID across all three "steps"

**Review by maxxcrawford — COMMENTED** [review]
Quick self-review

File: en/app.ftl [hunk 2]
```diff
@@ -476,7 +503,7 @@ modal-delete-domain-address-warning-upgrade = If you use this alias to log in to
     If you recreate a deleted alias, emails sent to the original alias will continue to be forwarded.
 modal-delete-confirmation = Yes, I want to delete this alias.
 modal-domain-register-good-news = Good News!
-modal-domain-register-warning = Remember, you are allowed to register one domain only for your account and is not possible to change your domain later.
+modal-domain-register-warning = Remember, you can only register one domain. You won’t be able to change it later.
```
Comment (maxxcrawford) [inline en/app.ftl:506]: TODO: Add new ID here

Comment (maxxcrawford) [reply en/app.ftl]: Added!

File: en/app.ftl [hunk 3]
```diff
@@ -460,11 +462,36 @@ onboarding-premium-domain-description = With a custom domain, you can make your
 onboarding-premium-reply-title = Reply to emails without giving away your real address
 onboarding-premium-reply-description = Need to reply to emails sent to an alias? Just reply as normal — your alias will still protect your email address.
 
+## Multi-part Premium Onboarding - This is a walk-through onboarding expereince
+multi-part-onboarding-premium-welcome-headline = Welcome to { -brand-name-relay-premium }
+multi-part-onboarding-premium-welcome-subheadline = Now you can control what hits your inbox, one email at a time.
+multi-part-onboarding-premium-welcome-title = Control what emails you get:
```
[the remark this answers is not in the document]
Comment (peiying2) [reply en/app.ftl:468]: We may have to repeat this one. The good thing is the Pontoon would prompt localizer that a string like this is already translated, and they can copy it over.

For some languages, there is no difference in translation. For others, the same string shows up as a bullet item vs part of a paragraph, the translation could be different - the verb. Let's keep what you have here. 

`Learn More` has to be the most repeated strings, but there is an ID for each use case.

File: en/app.ftl [hunk 4]
```diff
@@ -460,11 +462,36 @@ onboarding-premium-domain-description = With a custom domain, you can make your
 onboarding-premium-reply-title = Reply to emails without giving away your real address
 onboarding-premium-reply-description = Need to reply to emails sent to an alias? Just reply as normal — your alias will still protect your email address.
 
+## Multi-part Premium Onboarding - This is a walk-through onboarding expereince
```
Comment (peiying2) [inline en/app.ftl:465]: Add a line space for section comment

File: en/app.ftl (hunk 3 above)
[the remark this answers is not in the document]
Comment (maxxcrawford) [reply en/app.ftl:468]: Got'cha. Thanks!

File: en/app.ftl (hunk 1 above)
Comment (maxxcrawford) [reply en/app.ftl:472]: I removed these and used the original strings.

File: en/app.ftl (hunk 4 above)
Comment (maxxcrawford) [reply en/app.ftl:465]: Added in 9731f8f389e67df60bf7cdfb43fa172f4f896355

**Pull request merged**